kfdef/kfctl_openshift.v1.1.0.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  name: kubeflow
+  namespace: kubeflow
+spec:
+  applications:
+# This needs to be first to set up permissions
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: stacks/openshift/application/openshift/openshift-scc
```

Comment:
should we change the `application` directory name to plural similar to `stacks`?

Reply:
Discard this comment, somehow the singular application is being used on other stacks as well.